rosdep/osx-homebrew.yaml

```diff
@@ -95,6 +95,10 @@ eigen:
     homebrew:
       depends: [gfortran]
       packages: [eigen]
+exiv2:
+  osx:
+    homebrew:
+      packages: [exiv2]
```

Comment:
We are moving more towards a model where we put these keys in the main `base.yaml` file.  So if you can move that there, that would be appreciated.

Reply:
If the homebrew rules should go in `base.yaml`, have you considered updating the contributing guidelines to say that? 
https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules

Reply:
Good point, I'll go ahead and do that.

Reply:
See https://github.com/ros/rosdistro/pull/37248